### PR TITLE
Run the CNI signature per host

### PR DESCRIPTION
**Pull request by omertuc**
The CNI signature used to run only on the first host. I've modified it
to look for the logs on all hosts rather than just the first one

### CODE REVIEW

File: tools/add_triage_signature.py [hunk 1]
```diff
@@ -503,6 +505,29 @@ class CNIConfigurationError(Signature):
     def __init__(self, jira_client):
         super().__init__(jira_client, comment_identifying_string="h1. No CNI configuration")
 
+    @staticmethod
+    def _get_host_remote_hosts_ip_addresses(host):
```
Comment (osherdp) [inline tools/add_triage_signature.py:509]: so just ``_get_neighbors``? or ``_get_host_neighbors``

File: tools/add_triage_signature.py [hunk 2]
```diff
@@ -503,6 +505,29 @@ class CNIConfigurationError(Signature):
...
+        """
+        Given a host object, return a list of all IP addresses of the remote hosts in this host's
+        connectivity stanza.
```
Comment (osherdp) [inline tools/add_triage_signature.py:512]: A bit of confusing documentation to be honest

File: tools/add_triage_signature.py [hunk 3]
```diff
@@ -503,6 +505,29 @@ class CNIConfigurationError(Signature):
...
+        """
+        try:
+            return [(remote_host['host_id'], remote_host['l3_connectivity'][0]['remote_ip_address'])
+                    for remote_host in
+                    json.loads(host["connectivity"])['remote_hosts']]
+        except Exception:
```
Comment (osherdp) [inline tools/add_triage_signature.py:518]: in which case we might have this exception?

Comment (omertuc) [reply tools/add_triage_signature.py:518]: KeyError mostly, malformed service output :man_shrugging:

**Review by osherdp — APPROVED** [review]

**Pull request merged**